pkg/cmd/cli/cmd/set/volume.go

```diff
@@ -609,6 +609,10 @@ func (v *VolumeOptions) addVolumeToSpec(spec *kapi.PodSpec, info *resource.Info,
 			}
 			spec.Volumes = append(spec.Volumes[:i], spec.Volumes[i+1:]...)
 			break
+		} else {
+			if opts.Overwrite {
```

Comment:
Maybe this could just be an info message (either through `glog`, or simply text appended to the end of the output). Also, maybe you should check to see that `v.Name` did not match any of the `spec.Volumes` before returning this message. Maybe something like:

```go
vNameFound := false
for i, vol := range spec.Volumes {
    if v.Name == vol.Name {
        vNameFound = true
    }
}

if !vNameFound && opts.Overwrite {
    // handle overwrite info when volume by name given does not exist
}
```